logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/EventReceiver.kt

```diff
@@ -20,8 +20,9 @@ package com.wire.kalium.logic.sync.receiver
 
 import com.wire.kalium.logic.CoreFailure
 import com.wire.kalium.logic.data.event.Event
+import com.wire.kalium.logic.data.event.EventDeliveryInfo
 import com.wire.kalium.logic.functional.Either
 
 internal fun interface EventReceiver<T : Event> {
-    suspend fun onEvent(event: T): Either<CoreFailure, Unit>
+    suspend fun onEvent(event: T, deliveryInfo: EventDeliveryInfo): Either<CoreFailure, Unit>
```

Comment:
Question: I commented before I came to this eventually. It's not clear to me why every `EventReceiver` implementation needs this `deliveryInfo` since it's not used 🤔  Maybe I'm overseeing something.